### PR TITLE
Nazar's markup (page-1 and page-3)

### PR DESCRIPTION
**Pull request by wriozumi**
Nazar's markup

- page-1
- page-3

### CODE REVIEW

File: teams/4/page-1.css [hunk 1]
```diff
@@ -925,6 +929,210 @@ ul li:not(:first-of-type):not(.company__logo) {
   }
 }
 
+/* sliderSection (Nazar) */
+
+.sliderSection {
```
Comment (iamcucusa) [inline teams/4/page-1.css:934]: for all the classes the proper naming convention is slider-section, avoid using this camel case

File: teams/4/page-1.css [hunk 2]
```diff
@@ -925,6 +929,210 @@ ul li:not(:first-of-type):not(.company__logo) {
...
+  cursor: pointer;
+  color: var(--color-white);
+}
+
+.sliderList {
+  display: flex;
+}
+
+.slick-track {
+  display: flex !important;
```
Comment (iamcucusa) [inline teams/4/page-1.css:966]: the use of important is extremely forbidden, please asap remove it all over

File: teams/4/page-1.html [hunk 3]
```diff
@@ -23,6 +23,9 @@
   <link rel="stylesheet" href="../../css/navigation.css" />
   <link rel="stylesheet" href="../../css/main.css" />
   <link rel="stylesheet" href="../../css/normalize.css" />
+  <link rel="stylesheet" type="text/css" href="slick/slick.css" />
+  <link rel="stylesheet" type="text/css" href="slick/slick-theme.css" />
+  <link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.css" />
```
Comment (iamcucusa) [inline teams/4/page-1.html:28]: what is this? we are not using any external styles, remove

File: teams/4/page-3.css [hunk 4]
```diff
@@ -15,6 +15,7 @@
 body {
```
Comment (iamcucusa) [inline teams/4/page-3.css:15]: do not styles using html elements

File: teams/4/page-3.css [hunk 5]
```diff
@@ -28,7 +29,300 @@ ol {
...
+}
+
+.desktop-header .wrap button {
+  padding: 10px 16px;
+  border-radius: 8px;
+  border: 1px solid var(--color-gray-300);
+  cursor: pointer;
+}
+
+main {
```
Comment (iamcucusa) [inline teams/4/page-3.css:79]: same, do not style using elements

File: teams/4/page-3.css [hunk 6]
```diff
@@ -28,7 +29,300 @@ ol {
...
+  margin-right: 12px;
+}
+
+.avatar .wrap {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 6px;
+}
+
+/* .message {
```
Comment (iamcucusa) [inline teams/4/page-3.css:143]: remove the comment

File: teams/4/page-3.css [hunk 7]
```diff
@@ -28,7 +29,300 @@ ol {
...
+
+.company-info img {
+  margin-right: 12px;
+}
+
+.company-info span {
+  display: block;
 }
 
 .mobile__header {
```
Comment (iamcucusa) [inline teams/4/page-3.css:328]: should be ut-header mobile

File: teams/4/page-1.css (hunk 2 above)
Comment (wriozumi) [reply teams/4/page-1.css:966]: @iamcucusa  here I'm overriding the slick-slider plugin's default styles. Without !important it is impossible to override styles.

File: teams/4/page-1.html (hunk 3 above)
Comment (wriozumi) [reply teams/4/page-1.html:28]: fixed in the team-4 branch

**Pull request merged**